pkg/acceptance/helpers/common.go

```diff
@@ -81,3 +71,12 @@ func AssertErrorContainsPartsFunc(t *testing.T, parts []string) resource.ErrorCh
 		return nil
 	}
 }
+
+// EnumToTerraformStringList converts a list of enum values to the format represented in Terraform errors.
```

Comment:
We already have:
```
func AsStringList[T ~string](input []T) []string {
	output := make([]string, len(input))
	for i, element := range input {
		output[i] = string(element)
	}
	return output
}
```
so please handle only the format part here.

Reply:
Also, for me, it's a bit too much to extract it as function, because it only follows the error format in `func StringInSlice`, right?

Reply:
Removed.